File: web/api/src/ERS.Application.Contracts/DTO/BDExp/AssignStep.cs
using System.Collections.Generic;
using ERS.Application.Contracts.DTO.EmployeeInfo;
namespace ERS.DTO
{
    public class AssignStep
    {
        public string name { get; set; }
        public string position { get; set; }
        public List<EmployeeInfoDto> approverList { get; set; }
    }
}

File: web/ui/src/app/shared/models/assign-step.ts
export interface AssignStep {
  name: string;
  position: string;
  approverList: Approver[];
}

export interface Approver {
  emplid: string; // 工號
  name: string; // 中文名
  nameA: string; // 英文名
  display: string; // 顯示Label 顯示為英文名(若無，則顯示中文名)
}
